src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -264,7 +272,22 @@
         limit-insufficient?                         (> (controlled-input/numeric-value input-state)
                                                        current-limit)
         should-try-again?                           (and (not limit-insufficient?) no-routes-found?)
-        current-address                             (rf/sub [:wallet/current-viewing-account-address])]
+        current-address                             (rf/sub [:wallet/current-viewing-account-address])
+        owned-native-token                          (rf/sub [:wallet/token-by-symbol
```

Comment:
Maybe we can name it `owned-eth-token`?

Reply:
In different parts of the app, it's called `native` because it can be changed to other assets, so going with `native` is better imo

Reply:
But if it can be changed from "ETH" to something else, probably we shouldn't leave the `ETH` hardcoded?

Reply:
The issue here is that the back end will send this to us when getting a route is successful. When there is no enough assets, the back end won't send it